docs/docs/architecture/observability.md

```diff
@@ -30,3 +30,24 @@ For the mesh API, the metric names are prefixed with `meshapi_grpc_server_`. The
 metrics include similar data to the user API for the method `NewMeshCert` which
 gets called by the [Initializer](../components#the-initializer) when starting a
 new workload.
+
+## Service Mesh metrics
+
+The [Service Mesh](../components/service-mesh.md) can be configured to expose
+metrics via its [Envoy admin
+interface](https://www.envoyproxy.io/docs/envoy/latest/operations/admin). Be
+aware that the admin interface can expose private information and allows
+destructive operations to be performed. To enable the admin interface for the
+Service Mesh, set the environment variable `EDG_ADMIN_PORT` in the configuration
+of the Service Mesh sidecar. If this variable is set, the admin interface will
+be started on this port.
+
+By default, the Service Mesh is configured to start the admin interface on port
```

Comment:
Is that true? The implementation seems to only configure the admin port if it's explicitly set.

Reply:
Envoy automatically starts an administration interface if an address is specified. If not specified, the server will not start, see [docs](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/bootstrap/v3/bootstrap.proto#envoy-v3-api-msg-config-bootstrap-v3-admin).